ansible/roles/edu-sharing/tasks/edu-sharing-docker/migrate-alfresco-data.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Create the 3_datamigration-common.yml for database migration.
+  template:
+    src: 3_datamigration-common.yml.j2
+    dest: "{{ edu_docker_inst_dir }}/3_datamigration-common.yml"
+    mode: "600"
+  tags:
+    - migration-alfresco-sql
+
+- name: Launch edu-sharing in docker.
+  command:
+    chdir: "{{ edu_docker_inst_dir }}"
+    cmd: "{{edu_sharing_deploy_command}}"
+  no_log: true
+
+- name: Get the path to the alfresco docker volume.
+  shell: 'sg docker -c "docker volume inspect --format ''{{ ''{{'' }}.Mountpoint{{ ''}}'' }}''  $(docker volume ls -q |grep ''_repository-service-volume-data'')" '
```

Comment:
```suggestion
  shell: 'sg docker -c "docker volume inspect --format ''{{ ''{{'' }}.Mountpoint{{ ''}}'' }}''  $(sg docker -c "docker volume ls -q |grep ''_repository-service-volume-data''")" '
```

Reply:
tested it locally in the Vagrant VM => this way, the migration from 6.x works

Reply:
Hello @mirjan-hoffmann 

Yes looks like the ansible (or maybe is the vagrant problem) will not update the current user session even though the plugin we are using `https://galaxy.ansible.com/geerlingguy/docker` is trying to refresh the user session by resetting `ssh`

```
- name: Reset ssh connection to apply users changes
  meta:
    reset_connection
```